build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in

```diff
@@ -267,40 +255,13 @@
         </ul>
       </description>
     </release>
-
-    <release version="2.2.1" date="2018-04-03"/>
-    <release version="2.2" date="2018-03-26"/>
-    <release version="2.1.0" date="2017-05-02"/>
+    <release date="2018-04-03" version="2.2.1"/>
+    <release date="2018-03-26" version="2.2"/>
+    <release date="2017-05-02" version="2.1.0"/>
   </releases>
-  
-  <content_rating type="oars-1.1">
-    <content_attribute id="violence-cartoon">none</content_attribute>
-    <content_attribute id="violence-fantasy">none</content_attribute>
-    <content_attribute id="violence-realistic">none</content_attribute>
-    <content_attribute id="violence-bloodshed">none</content_attribute>
-    <content_attribute id="violence-sexual">none</content_attribute>
-    <content_attribute id="violence-desecration">none</content_attribute>
-    <content_attribute id="violence-slavery">none</content_attribute>
-    <content_attribute id="violence-worship">none</content_attribute>
-    <content_attribute id="drugs-alcohol">none</content_attribute>
-    <content_attribute id="drugs-narcotics">none</content_attribute>
-    <content_attribute id="drugs-tobacco">none</content_attribute>
-    <content_attribute id="sex-nudity">none</content_attribute>
-    <content_attribute id="sex-themes">none</content_attribute>
-    <content_attribute id="sex-homosexuality">none</content_attribute>
-    <content_attribute id="sex-prostitution">none</content_attribute>
-    <content_attribute id="sex-adultery">none</content_attribute>
-    <content_attribute id="sex-appearance">none</content_attribute>
-    <content_attribute id="language-profanity">none</content_attribute>
-    <content_attribute id="language-humor">none</content_attribute>
-    <content_attribute id="language-discrimination">none</content_attribute>
-    <content_attribute id="social-chat">none</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
-    <content_attribute id="social-audio">none</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
-    <content_attribute id="social-contacts">none</content_attribute>
-    <content_attribute id="money-purchasing">none</content_attribute>
-    <content_attribute id="money-gambling">none</content_attribute>
-  </content_rating>
-
-</application>
+  <provides>
+    <id>mscore.desktop</id>
+  </provides>
+  <launchable type="desktop-id">mscore.desktop</launchable>
```

Comment:
The desktop file is actually called `mscore${MSCORE_INSTALL_SUFFIX}.desktop`. The CMake variable `${MSCORE_INSTALL_SUFFIX}` is empty by default, but developers can set it to enable them to install more than one version at a time.